pallets/phala/src/fat.rs

```diff
@@ -103,12 +108,64 @@ pub mod pallet {
 
 	type CodeHash<T> = <T as frame_system::Config>::Hash;
 
+	fn check_cluster_permission<T: Config>(
+		deployer: &T::AccountId,
+		permission: &ClusterPermission<T::AccountId>,
+	) -> bool {
+		match permission {
+			ClusterPermission::Public => true,
+			ClusterPermission::OnlyOwner(owner) => deployer == owner,
+		}
+	}
+
 	#[pallet::call]
 	impl<T: Config> Pallet<T>
 	where
 		T: crate::mq::Config + crate::registry::Config,
 		T::AccountId: AsRef<[u8]>,
 	{
+		#[pallet::weight(0)]
+		pub fn add_cluster(
+			origin: OriginFor<T>,
+			permission: ClusterPermission<T::AccountId>,
+			description: String,
+			deploy_workers: Vec<WorkerPublicKey>,
+		) -> DispatchResult {
+			// for now, we only allow root account to create cluster
```

Comment:
```suggestion
			// For now, we only allow root account to create cluster
```

It's suggested to write formal sentences in comments